### ISSUE TITLE
New Relic for non profits

### ISSUE THREAD

**Issue by snufkin** [drupalhu/drupal.hu#171]
a New Relic ajanl non profit szervezeteknek ingyenes pro elofizetest (http://newrelic.com/nonprofit), amire ha meglesz az alapitvany vegre, szerintem mi jelentkezhetnenk. A kiiras szerint az 501(c)(3) kategoriabol jelentkezhetnek ra (http://en.wikipedia.org/wiki/501%28c%29_organization).

**Comment by tanarurkerem** [comment]
igen, ránk nem vonatkozik ez

aboros [9:44 AM]
sztem

aboros [9:45 AM]
bár...

aboros [9:45 AM]
istvan.palocz: 501c3 mindent felsorol kb..

aboros [9:45 AM]
nem csak alapitvanyt

aboros [9:47 AM]
de nem, a célok is adottak.. talán educational purposes?

aboros [9:47 AM]
:simple_smile:

aboros [9:47 AM]
vagy religion (drupal!) :simple_smile:

tompagabor [9:47 AM] 
Nemzetközi jogban ki van otthon :simple_smile: kéne egy ügyvéd ehez :simple_smile: Drifter faterja elvileg keni-vágja meg tanította is a jogot, de nem tudom, hogy a jog ezen témakörét is-e

istvan.palocz [9:53 AM] 
egyszerűbb, hogy ha leírjuk a newrelicnek a story-t meg, hogy lesz egy olyan showcase, ahol bemutatjuk, hol és mire használjuk, meg van egy párezres közösség aki ezt olvassa, talán akkor érdekli őket.

**Comment by Sweetchuck** [comment]
Végül ez hogyan oldódott meg? Ha jól látom akkor a PROD környezeten be van állítva a New Relic.

**Comment by snufkin** [comment]
Igen, de csak az alap subscription, ami nem mutat pl fuggveny stack-et
